resources/views/components/post-item.blade.php

```diff
@@ -7,6 +7,39 @@
         </div>
 
         <div class="flex space-x-2">
+            @if (\Firefly\Features::enabled('correct_posts'))
+                @if ($post->is_correct)
+                    @can ('unmark', $post)
+                        <button onclick="event.preventDefault(); document.getElementById('unmark-post-{{ $post->id }}-form').submit();">
+                    @endcan
+                        <span class="sr-only">{{ __('Mark as Incorrect') }}</span>
+                        <svg class="w-5 h-5 text-gray-700 hover:text-gray-500"  @can ('unmark', $post) data-tippy-content="{{ __('Unmark as Correct') }}" @else data-tippy-content="{{ __('This is Correct') }}" @endcan xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20" fill="currentColor" aria-hidden="true">
+                            <path fill-rule="evenodd" d="M6.267 3.455a3.066 3.066 0 001.745-.723 3.066 3.066 0 013.976 0 3.066 3.066 0 001.745.723 3.066 3.066 0 012.812 2.812c.051.643.304 1.254.723 1.745a3.066 3.066 0 010 3.976 3.066 3.066 0 00-.723 1.745 3.066 3.066 0 01-2.812 2.812 3.066 3.066 0 00-1.745.723 3.066 3.066 0 01-3.976 0 3.066 3.066 0 00-1.745-.723 3.066 3.066 0 01-2.812-2.812 3.066 3.066 0 00-.723-1.745 3.066 3.066 0 010-3.976 3.066 3.066 0 00.723-1.745 3.066 3.066 0 012.812-2.812zm7.44 5.252a1 1 0 00-1.414-1.414L9 10.586 7.707 9.293a1 1 0 00-1.414 1.414l2 2a1 1 0 001.414 0l4-4z" clip-rule="evenodd" />
+                        </svg>
+                    @can ('unmark', $post)
+                        </button>
+                        
+                        <form id="unmark-post-{{ $post->id }}-form" action="{{ route('firefly.post.incorrect', $post) }}" method="POST" style="display: none;">
+                            @method('DELETE')
+                            @csrf
+                        </form>
+                    @endcan
+                @else
```

Comment:
Rather than so much if/else/can/else, we could probably just split it into two different if/else?

Reply:
I could also abandon the @can directive, and just use the logic directly in the if:
`if ($post->is_correct && auth()->user()->can(...) `

Reply:
I merged in your suggested changes